Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -234,20 +270,22 @@ class AttachmentSaveEventListener final : public EventListener {
 
 void HTMLAttachmentElement::updateSaveButton(bool show)
 {
-    if (!m_containerElement)
-        return;
-
     if (!show) {
         if (m_saveButton) {
-            m_containerElement->removeChild(*m_saveButton);
+            m_informationBlock->removeChild(*m_saveArea);
             m_saveButton = nullptr;
+            m_saveArea = nullptr;
         }
         return;
     }
 
-    if (!m_saveButton) {
-        m_saveButton = createContainedElement<HTMLButtonElement>(*m_containerElement, attachmentSaveButtonIdentifier(), String { });
+    if (!m_saveButton && m_titleElement) {
```

Comment:
Wonder if this should also be `!m_saveArea`.

Reply:
Same-same, as they have the same lifetime. I prefer to mention the button here, as it's the most important element, the area is just an css-implementation detail.